api/utils/image-processor.js

```
@@ -0,0 +1,84 @@
+    });
+  }
+  
+  // Apply format conversion with appropriate quality
+  switch (format) {
+    case 'webp':
+      pipeline = pipeline.webp({ 
+        quality: quality || WEBP_QUALITY,
+        effort: 4
+      });
+      break;
+    case 'jpeg':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)